superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -323,11 +325,9 @@ def _benchmark(self):
                 self._sub_benchmark_start_time = time.time()
                 if model_action == ModelAction.TRAIN:
                     if not self.__train(precision):
-                        self._result.set_return_code(ReturnCode.MODEL_TRAIN_FAILURE)
                         return False
                 elif model_action == ModelAction.INFERENCE:
                     if not self.__inference(precision):
-                        self._result.set_return_code(ReturnCode.MODEL_INFERENCE_FAILURE)
```

Comment:
Do we need to remove ReturnCode.MODEL_INFERENCE_FAILURE and ReturnCode.MODEL_TRAIN_FAILURE in definition?

Reply:
Sure, I will remove them.